jsettlers.mapcreator/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+encoding//src/jsettlers/mapcreator/localization/labels_de.properties=UTF-8
+encoding//src/jsettlers/mapcreator/localization/labels_en_US.properties=UTF-8
+encoding//src/jsettlers/mapcreator/localization/labels_ru.properties=UTF-8
```

Comment:
It seems like this isn't possible, as Eclipse always interprets .properties files with their Java default encoding. It should be possible when we switch to another ending.